contacts/views.py

```diff
@@ -82,3 +85,45 @@ def post(self, request, pk):
         contact = Contact.objects.get(id=pk)
         contact.delete()
         return HttpResponseRedirect(reverse('contacts:contact'))
+
+
+class LoginView(View):
```

Comment:
all these views belong to user not contact.
create a separate app for users and move these all over there!